services/web/assets/themes/opencloud/theme.json

```diff
@@ -167,46 +156,7 @@
               "icon-pdf": "#ec0d47",
               "icon-presentation": "#ee6b3b",
               "icon-spreadsheet": "#15c286",
-              "icon-video": "#045459",
```

Comment:
Do we really want to remove this ?

Reply:
It's not being removed, it's just being listed a new line because the comma at the end has been removed.